scripts/aggregate_result.py

```diff
@@ -52,9 +52,13 @@ def get_average_score(input_path: str, keys: List[str]) -> float:
     return sum(scores) / len(scores)
 
 
-def aggregate_results(model: str) -> Dict[str, float]:
+def aggregate_results(
+    model: str,
+    result_dir: str = None,
+    column_path_key_path: str = "scripts/column-path-key.csv",
```

Comment:
```suggestion
    result_dir: str,
    column_path_key_path: str,
```

外からのargument passingがあるので、default valueが必要ないと思います。